### ISSUE TITLE
Can GIF encoding be offloaded to a web worker?

### ISSUE THREAD

**Issue by sole** [sole/Animated_GIF#14]
Currently the frames are processed in workers, but why is the encoding not in a worker?

**Comment by Montoya** [comment]
Uneducated guess: if encoding creates a blob, that blob is stored in memory. If it happens in a worker, that blob is in the memory space of the worker, but is not accessible to the main thread.

**Comment by sole** [comment]
@Montoya, that was a rhetoric question.

Additionally you can transfer object ownership between workers and the main thread.

**Comment by Montoya** [comment]
Yes I realized that like two minutes after posting it (and that the question was over a year old) but figured it was too late :(
